src/main/java/com/jezhumble/javasysmon/LinuxMonitor.java

```diff
@@ -66,17 +71,27 @@ public String osName() {
     }
 
     public MemoryStats physical() {
-        String totalMemory = fileUtils.runRegexOnFile(TOTAL_MEMORY_PATTERN, "/proc/meminfo");
+        String totalMemory = fileUtils.runRegexOnFile(TOTAL_MEMORY_PATTERN, PROC_MEMINFO);
         long total = Long.parseLong(totalMemory) * 1024;
-        String freeMemory = fileUtils.runRegexOnFile(FREE_MEMORY_PATTERN, "/proc/meminfo");
+        String freeMemory = fileUtils.runRegexOnFile(FREE_MEMORY_PATTERN, PROC_MEMINFO);
         long free = Long.parseLong(freeMemory) * 1024;
         return new MemoryStats(free, total);
     }
 
+    public MemoryStats physicalWithBuffersAndCached(){
+        String totalMemory = fileUtils.runRegexOnFile(TOTAL_MEMORY_PATTERN, PROC_MEMINFO);
+        long total = Long.parseLong(totalMemory) * 1024;
+        String freeMemory = fileUtils.runRegexOnFile(FREE_MEMORY_PATTERN, PROC_MEMINFO);
+        String buffers = fileUtils.runRegexOnFile(BUFFERS_PATTERN, PROC_MEMINFO);
+        String cached = fileUtils.runRegexOnFile(CACHED_PATTERN, PROC_MEMINFO);
+        long free = Long.parseLong(freeMemory) * 1024 + Long.parseLong(buffers) * 1024 + Long.parseLong(cached) * 1024;
```

Comment:
Maybe it would be better:

`long free = (Long.parseLong(freeMemory) + Long.parseLong(buffers) + Long.parseLong(cached)) * 1024;`

So only it is needed to multiply * 1024 once to convert to bytes.